doc/RELEASE_NOTES.md

```diff
@@ -84,62 +87,51 @@ While this is not a new feature of the Pbench Agent, it is worth noting that whe
 
 ## All Tool Registration Handled Locally
 
-Along with the new "Tool Meister" sub-system comes another subtle, but significant, change to how tools are registered.
+Along with the new "Tool Meister" sub-system comes a subtle, but significant, change to how tools are registered.
 
-With the v0.71 release, the record of which tools are registered on which hosts is kept local to the host on which `pbench-register-tool` or `pbench-register-tool-set` are invoked.
+Prior to v0.71, tool registration for remote hosts was recorded locally, _and also_ remotely via ssh.
 
-Prior to v0.71, tool registration for remote hosts was recorded locally, and remotely via ssh.
+With v0.71, tools are recorded only locally when they are registered and the validation of remote hosts is deferred until the workload is run. During its initialization, the Tool Meister sub-system now reports when registered tools are not present on registered hosts, and, if a tool is not installed, an error message will be displayed, and the "bench-script" will exit with a failure code.
 
 The registered tools are recorded in a local directory off of the "pbench_run" directory, by default `/var/lib/pbench-agent/tools-v1-<name>`, where `<name>` is the name of the Tool Group under which the tools were registered.
 
-The process of registering tools on local or remote hosts no longer validates that those tools are available during tool registration.  The Tool Meister sub-system now reports when registered tools are not present on registered hosts before beginning a benchmark run.  An error message will be displayed, and the particular "bench-script" will exit with a failure code.
+The process of registering tools on local or remote hosts no longer validates that those tools are available during tool registration.  The Tool Meister sub-system now reports when registered tools are not present on registered hosts before beginning a benchmark workload.  An error message will be displayed, and the particular "bench-script" will exit with a failure code.
```

Comment:
The paragraph at line 98 appears to be substantially a duplicate of the one at line 94.